src/main/java/thePackmaster/events/BlackMarketDealerEvent.java

```diff
@@ -144,21 +152,23 @@ public void update() {
                             AbstractDungeon.effectList.add(new ShowCardAndObtainEffect(new CurseOfTheBell(), (Settings.WIDTH * .33F), (float) (Settings.HEIGHT / 2)));
                             //AbstractDungeon.effectList.add(new ShowCardAndObtainEffect(new CurseOfTheBell(), (Settings.WIDTH * .75F), (float) (Settings.HEIGHT / 2)));
                             AbstractDungeon.effectList.add(new ShowCardAndObtainEffect(new ConjurePack(), (Settings.WIDTH * .66F), (float) (Settings.HEIGHT / 2)));
+                            logMetricObtainCards(ID, "Conjure Pack", Arrays.asList(ConjurePack.ID, CurseOfTheBell.ID));
                             transitionKey("magicLearnEnd");
                         }).addOption(OPTIONS[12], (i) -> {   //Pack in a Jar Potion
                             AbstractDungeon.getCurrRoom().rewards.clear();
                             AbstractDungeon.getCurrRoom().rewards.add(new RewardItem(new PackInAJar()));
                             skipDefaultCardRewards = true;
                             AbstractDungeon.combatRewardScreen.open();
                             skipDefaultCardRewards = false;
+                            logMetric(ID, "Pack in a Jar", null, null, null, null, null, Collections.singletonList(PackInAJar.POTION_ID), null, 0, 0, 0, 0, 0, 0);
```

Comment:
There are a few oddities with the event choices like this one on the run history page -- the base game isn't set up to properly display event choices when both gain a relic and gain cards/potions, and the patch to fix that is tricky enough that I decided against doing it right now. You can still work out what happened from what run history displays, it's just missing some information.